doc/introduction/circuits.rst

```diff
@@ -94,37 +85,73 @@ instantiated using the :func:`device <pennylane.device>` loader.
 
     dev = qml.device('default.qubit', wires=2, shots=1000)
 
-PennyLane offers some basic devices such as the ``'default.qubit'`` and ``'default.gaussian'``
-simulators; additional devices can be installed as plugins (see
+PennyLane offers some basic devices such as the ``'default.qubit'``, ``'default.mixed'``
+and ``'default.gaussian'`` simulators; additional devices can be installed as plugins (see
```

Comment:
```suggestion
PennyLane offers some basic devices such as the ``'default.qubit'``, ``'default.mixed'``, ``lightning.qubit``,
and ``'default.gaussian'`` simulators; additional devices can be installed as plugins (see
```

Reply:
But lightning.qubit does not yet ship with PL? It is not in the `device` folder at least?